docs/source/theory/intro.rst

```diff
@@ -74,18 +74,86 @@ The first option is to characterize the distribution via the intersections of th
 .. math::
 
    \begin{align}
-        \sigma_q &= \sqrt{\frac{\epsilon}{\gamma}} \\
-        \sigma_p &= \sqrt{\frac{\epsilon}{\beta}} \\
+        \lambda_q &= \sqrt{\frac{\epsilon}{\gamma}} \\
+        \lambda_p &= \sqrt{\frac{\epsilon}{\beta}} \\
         \mu_{qp} &= \frac{\alpha}{\sqrt{\beta \gamma}}
    \end{align}
 
-The units are :math:`[\sigma_q] = \mathrm{m}`, :math:`[\sigma_p] = \mathrm{rad}`, and :math:`[\mu_{qp}] = 1`.
+The units are :math:`[\lambda_q] = \mathrm{m}`, :math:`[\lambda_p] = \mathrm{rad}`, and :math:`[\mu_{qp}] = 1`.
 To convert between normalized and unnormalized emittance, use the relation :math:`\epsilon_\mathrm{n} = (\beta\gamma)_\mathrm{ref} \cdot \epsilon` which uses the momentum of the reference particle.
 **Attention**: Here, :math:`(\beta\gamma)_\mathrm{ref}` are the Lorentz variables for the reference particle momentum and not the Courant-Snyder parameters.
 
 The second option is to specify the distribution via the Courant-Snyder / Twiss parameters :math:`\alpha` and :math:`\beta` along with the unnormalized (geometric, 1-RMS) emittance :math:`\epsilon` for all the spatial coordinates.
 Recall the Courant-Snyder relation :math:`\gamma\beta - \alpha^2 = 1` for conversion from :math:`\gamma` values to our input conventions.
 
+Distribution Sampling and the Covariance Matrix
+-----------------------------------------------
+
+Let :math:`P` denote a phase space probability density with unit covariance matrix (i.e., equal to the identity matrix).  To produce a phase space density with a target covariance matrix :math:`\Sigma`, we write :math:`\Sigma` in terms of its (lower) Cholesky decomposition as:
```

Comment:
Symbols I cannot find the introduction for yet:
- kappa
- zeta
- L
- G
- TS

Reply:
Introduced kappa, zeta, and L explicitly.  Looking for for G and TS.

Reply:
Thanks!

TS is used in `\zeta^TS\zeta`.
G in `P(\zeta)=G(||\zeta||^2)=G(q^2+p^2),\quad\quad \zeta=(q,p)`